cmd/nerdctl/restart_linux_test.go

```diff
@@ -43,6 +43,12 @@ func TestRestart(t *testing.T) {
 	newInspect := base.InspectContainer(tID)
 	newPid := newInspect.State.Pid
 	assert.Assert(t, pid != newPid)
+
+	base.Cmd("restart", "-t", "5", tID).AssertOK()
+	base.EnsureContainerStarted(tID)
+	timeInspect := base.InspectContainer(tID)
+	timePid := timeInspect.State.Pid
+	assert.Assert(t, pid != timePid)
```

Comment:
You are test does not highlight the grace period

Reply:
SGTM